packages/core/src/Tabs/TabsRoot.vue

```
@@ -104,8 +104,8 @@ provideTabsRootContext(context)
   <Primitive
     :as="as"
     :as-child="asChild"
-    v-bind="mergeProps(root.props.value, stateToDataAttrs(root.state.value))"
+    v-bind="root.attrs.value"
   >
-    <slot :model-value="modelValue" />
+    <slot :model-value="modelValue as T | undefined" />
```

ESLint

[error] Filters are deprecated.

(vue/no-deprecated-filter)